Update automation-create-runas-account.md

masterbacon opened the pull request:
Made alterations to the New-RunAsAccount.ps1 script due to changes in latest AzureRM powershell modules (v4.4.1).

The following object no longer exists:
Microsoft.Azure.Commands.Resources.Models.ActiveDirectory.PSADKeyCredential

Therefore, the key credential variables should be supplied as parameters to the New-AzureRMADServicePrincipal CmdLet.

Note - Not sure if $KeyId is still required, but left it in due to it being used as the -IdentifierUris parameter to the New-AzureRmADApplication CmdLet.

See discussion https://github.com/Azure/azure-powershell/issues/4491 for how I arrived at my solution.

azurecla commented:
Hi __@mingfeiy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (yanmf). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

mingfeiy commented:
I am testing my writing permission with Julia. Its a small change - could you approve?

masterbacon committed:
Update automation-create-runas-account.md

Made alterations to the New-RunAsAccount.ps1 script due to changes in latest AzureRM powershell modules (v4.4.1).

The following object no longer exists:
Microsoft.Azure.Commands.Resources.Models.ActiveDirectory.PSADKeyCredential

Therefore, the key credential variables should be supplied as parameters to the New-AzureRMADServicePrincipal CmdLet.

Note - Not sure if $KeyId is still required, but left it in due to it being used as the -IdentifierUris parameter to the New-AzureRmADApplication CmdLet.

See discussion https://github.com/Azure/azure-powershell/issues/4491 for how I arrived at my solution.
